sphinx/source/conf.py

```diff
@@ -64,16 +64,8 @@
 # built documents.
 #
 # Let's try to automatically get the version
-from bokeh._version import get_versions
-from bokeh import settings
-
-try:
-    from bokeh.__conda_version__ import conda_version
-    __version__ = conda_version.replace("'","")
-    del conda_version
-except ImportError:
-    __version__ = get_versions()['version']
-    del get_versions
+from bokeh import __version__
```

Comment:
Why you are deleting this from `conf.py`, it automatically get the correct version... it is important from the devel docs point of view...